raiden/api/python.py

```diff
@@ -193,25 +193,25 @@ def get_transfer_from_task(
 
 
 def transfer_tasks_view(
-    transfer_tasks: Dict[SecretHash, TransferTask],
+    payment_mapping: Dict[Address, PaymentMappingState],
```

Comment:
i think t's ok to keep this `Dict` type alias around since it was already present, but for a new structure, i'd always opt for native types rather than custom classes unless we need something specific from them.

just something to keep in mind